optaplanner-core/src/main/java/org/optaplanner/core/config/score/director/ScoreDirectorFactoryConfig.java

```diff
@@ -76,10 +76,9 @@
     @XmlElement(name = "scoreDrlFile")
     protected List<File> scoreDrlFileList = null;
     @XmlTransient
-    protected KieBaseExtractor kieBaseExtractor = null;
+    protected KieRuntimeBuilderWrapper gizmoKieRuntimeBuilderWrapper = null;
```

Comment:
I strongly believe we need to prefix this property with "gizmo", such that users that use code completion on this class  (when using programmatic API instead of XML) don't mistakenly believe that it's a typical property that they can modify to suit their needs.
By prefixing it with "gizmo" they clearly see that it's for gizmo's usage. As for other usages outside gizmo? YAGTNI.

Reply:
I was thinking we might want to make a GizmoConfig, as a property on SolverConfig and put all the gizmo related properties in there. But that would probably be a bad idea, as we want the score director related properties to sit close to the other code for that, so in ScoreDirectorFactoryConfig.
So instead, let's just all prefix them with gizmo (which we already do for the memberaccessor and solutioncloner)